src/eko/scale_variations/exponentiated.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+r"""
+This module contains the scale variation for ``ModSV=exponentiated``.
+"""
+import numba as nb
+
+from .. import beta
+
+
+@nb.njit(["c16[:,:,:](c16[:,:,:],u1,u1,f8)", "c16[:](c16[:],u1,u1,f8)"], cache=True)
+def gamma_variation(gamma, order, nf, L):
+    """
+    Adjust the anomalous dimensions with the scale variations.
+
+    Parameters
+    ----------
+        gamma : numpy.ndarray
+            anomalous dimensions
+        order : int
+            perturbation order
+        nf : int
+            number of active flavors
+        L : float
+            logarithmic ratio of factorization and renormalization scale
+
+    Returns
+    -------
+        gamma : numpy.ndarray
+            adjusted singlet anomalous dimensions
```

Comment:
Why here only the singlet is varied, and in the `expanded` both are?

Moreover, if it's only singlet, maybe the name of the function should reflect it.

Reply:
No it's a typo, it should be: `adjusted anomalous dimensions`. Thanks for spotting.

Reply:
Ok, perfect :)